proto/v1beta1/intoto.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package grafeas.v1beta1.intoto;
+
+message SignatureObject {
+  string keyid = 1;
+
+  string signature = 2;
+}
+
+message Signature {
+  repeated SignatureObject signatures = 1;
+}
+
+message ArtifactHashObject {
+  map<string, string> hashes = 1;
+}
+
+message Signed {
+  string _type = 1;
+
+  string name = 2;
+
+  repeated string command = 3;
+
+  map<string, ArtifactHashObject> materials = 4;
```

Comment:
Could you add more information for each of these fields, `materials`, `products` etc with clarifications on what they represent?

Reply:
I add comments for all the fields and included a link to the specification which goes into greater detail.